api/handler/put.go

```diff
@@ -551,7 +551,8 @@ func (h *handler) CreateBucketHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	h.log.Info("bucket is created",
-		zap.String("container_id", cid.String()))
+		zap.String("container_id", cid.String()),
+		zap.String("bucket name", p.Name))
```

Comment:
Do not use spaces in keys of zap's key-value pairs. It is a bit ruining of the ouput in JSON format, so try to avoid them.